src/Element.cc

```diff
@@ -1311,12 +1285,7 @@ void Element::RemoveChild(ElementPtr _child)
 /////////////////////////////////////////////////
 void Element::RemoveChild(ElementPtr _child, sdf::Errors &_errors)
```

Comment:
This should avoid the compiler warning
```suggestion
void Element::RemoveChild(ElementPtr _child, sdf::Errors &)
```